components/Comments.jsx

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from "react";
+import { getCommentsByArticleId } from "../src/api";
+import CommentCard from "./CommentCard";
+
+export default function Comments({ article_id }) {
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getCommentsByArticleId(article_id).then((comment) => {
+      console.log(comment);
+      setComments(comment);
+      setIsLoading(false);
+    });
+  }),
```

Comment:
you haven't set this `useEffect / dependency array` up correctly which means it is continuously rendering (check the console)